content/workshops/jboss-migration-toolkit/lab5-using-jmt-web-console.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Lab 5 - Using RHMAT Web Console
+workshops: jboss-migration-toolkit
+workshop_weight: 50
+layout: lab
+---
+
+# About Red Hat Application Migration Toolkit CLI
+The web console for Red Hat Application Migration Toolkit is a web-based system that allows a team of users to assess and prioritize migration and modernization efforts for a large number of applications. It allows you to group applications into projects for analysis and provides numerous reports that highlight the results. 
+
+{{% alert warning %}}
+The following instructions are only when you are doing this lab outside of the controlled workshop.  The workshop environment has downloaded, installed, and configured your student environments prior to commencement of this workshop session.
+{{% /alert %}}
+
+# Set Up
+1. Install the Java SE Development Kit (JDK) version 8. We recommend using the OpenJDK or the Oracle JDK.
+2. Install the RHAMT web console.
+  a. Download the [ZIP archive][1].
```

Comment:
Similar comment regarding the numbering and indenting. The rendered numbering is messed up throughout this page.

Take a look at the generated content on your local machine by running the command `hugo server`